deploy-manage/security/secure-your-cluster-deployment.md

```diff
@@ -9,67 +9,62 @@ applies_to:
-- **Remote cluster access**: Secure cross-cluster operations.
+:::{include} /deploy-manage/security/_snippets/cluster-communication-network.md
+:::
 
-Refer to [](traffic-filtering.md).
+## Data security
 
+:::{include} /deploy-manage/security/_snippets/cluster-data.md
+:::
+ 
+## User session security
 
-## Cluster communication
+:::{include} /deploy-manage/security/_snippets/cluster-user-session.md
+:::
 
-- **HTTP and HTTPs**
-- **TLS certificates and keys**
+## Security event audit logging
 
+:::{include} /deploy-manage/security/_snippets/audit-logging.md
+:::
 
-## Data, objects and settings security
+## Configure security in a self-managed cluster
 
-- **Bring your own encryption key**: Use your own encryption key instead of the default encryption at rest provided by Elastic.
-- **{{es}} and {{kib}} keystores**: Secure sensitive settings using keystores
-- **{{kib}} saved objects**: Customize the encryption for {{kib}} objects such as dashboards.
-- **{{kib}} session management**: Customize {{kib}} session expiration settings.
+Since {{es}} 8.0, security is enabled and configured by default. However, security auto configuration [might be skipped](/deploy-manage/security/manually-configure-security-in-self-managed-cluster.md#stack-skip-auto-configuration) in certain scenarios. In these cases, you can [manually configure security](/deploy-manage/security/manually-configure-security-in-self-managed-cluster.md).
 
-Refer to [](data-security.md).
+## FIPS 140-2 compliant mode
+```{applies_to}
+deployment:
+  self:
+  eck:
+```
+```
 
-## User roles
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB 140-2), titled "Security Requirements for Cryptographic Modules" is a U.S. government computer security standard used to approve cryptographic modules. You can run a self-managed cluster or {{eck}} cluster in FIPS-compliant mode:
 
-[Define roles](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md) for your users and [assign appropriate privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md) to ensure that users have access only to the resources that they need. This process determines whether the user behind an incoming request is allowed to run that request.
+* [Self-managed](/deploy-manage/security/fips-140-2.md)
+* [ECK](/deploy-manage/deploy/cloud-on-k8s/deploy-fips-compatible-version-of-eck.md)
 
-:::{important}
-Never try to run {{es}} as the `root` user, which would invalidate any defense strategy and permit a malicious user to do **anything** on your server. You must create a dedicated, unprivileged user to run {{es}}. By default, the `rpm`, `deb`, `docker`, and Windows packages of {{es}} contain an `elasticsearch` user with this scope.
-:::
+% we need to refine this table, but the idea is awesome IMO
+
+## Security features by deployment type [comparison-table]
 
+:::{include} /deploy-manage/security/_snippets/cluster-comparison.md
+:::
```

Comment:
I'm not seeing any of this in the preview!! I think due to a markdown issue earlier that I'll try to correct.